accel/tcg/log_instr.c

```diff
@@ -233,7 +233,9 @@ typedef struct {
 #define CTE_QEMU_MAGIC      "CheriTraceV03"
 
 /* Number of per-cpu ring buffer entries for ring-buffer tracing mode */
-#define QEMU_LOG_INSTR_CPU_BUF_ENTRIES (1 << 16)
+#define MIN_ENTRY_BUFFER_SIZE (1 << 16)
```

Comment:
It seems like you can also shrink the buffer at runtime, maybe this should be default_ not min_?

Reply:
qemu_log_instr_set_buffer_size enforces this?

Reply:
Oops I only looked at do_log_buffer_resize(CPUState